### PR TITLE
look for annotations in the cache

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: BaseApi/EventSubscriber/AbstractSubscriber.php [hunk 1]
```diff
@@ -59,6 +48,6 @@ protected function extractAnnotation(KernelEvent $event, $annotationClassName)
      */
     protected function eventElligible(KernelEvent $event)
     {
-        return $event->isMasterRequest() && $this->isApiRequest($event->getRequest());
```
Comment (ngilain) [inline BaseApi/EventSubscriber/AbstractSubscriber.php:62]: So no more check ?

Comment (nicolasThal) [inline BaseApi/EventSubscriber/AbstractSubscriber.php:62]: Yes, no more check.

As we use the cacheReader for the annotations, the impact is less important.

The main reason for this change is to allow every integartor to create it's own api without taking care of the route name

**Comment by juchi** [comment]
:+1:  thanks !

File: BaseApi/EventSubscriber/AbstractSubscriber.php (hunk 1 above)
Comment (ngilain) [inline BaseApi/EventSubscriber/AbstractSubscriber.php:62]: Fine

**Comment by ngilain** [comment]
:+1:

**Pull request merged**